client/mds3/skewer.render.js

```diff
@@ -636,9 +644,17 @@ export function unfold_glyph(newlst, tk, block) {
 				d.y = d.yoffset * (tk.skewer.pointup ? -1 : 1)
 				return 'translate(0,' + d.y + ')'
 			})
-		setTimeout(function () {
-			set.selectAll('.sja_aa_disckick').attr('transform', 'scale(1)')
-		}, dur)
+		// setTimeout(function () {
+		set
+			.selectAll('.sja_aa_disckick')
+			.transition()
+			.duration(dur)
+			.attr('transform', 'scale(1)')
+			.on('end', () => {
+				//For e2e testing
+				set.selectAll('.sja_aa_disckick').classed('active', true)
```

Comment:
change simple class name 'active' to 'sjpp-active' to avoid potential conflict with portal?